src/main/java/se/kth/jabeja/Jabeja.java

```diff
@@ -41,19 +52,20 @@ public void startJabeja() throws IOException {
       //one cycle for all nodes have completed.
       //reduce the temperature
       saCoolDown();
```

Comment:
Should we call this even when not running SA? I thought we then wanted linear evolution of T

Reply:
You're right!